### PR TITLE
Fix tag background

### PR DESCRIPTION
**Pull request by AndrewMusgrave**
### WHY are these changes introduced?

Card/Modal/Tag backgrounds are all white. Let's align them to [figma](https://www.figma.com/file/4dAAt5iFPSaxUKiYVKrkYj/Admin-Design-Language-Web-Components?node-id=364%3A1478)

Interactions need to write for a release of polaris tokens to be updated. Issue [here](https://github.com/Shopify/polaris-react/issues/3352) so we don't forget about it.

### CODE REVIEW

**Comment by kyledurand** [comment]
Those states should be available now. They were merged [here](https://github.com/Shopify/polaris-tokens/pull/155/files) and made available in 2.14.0. Let me know if they're not present and I'll look into it

**Comment by AndrewMusgrave** [comment]
Awesome that was super quick, just updated them 🙌

File: src/components/Tag/README.md [hunk 1]
```diff
@@ -39,6 +39,14 @@ Use to allow merchants to add attributes to, and remove attributes from, an obje
 <Tag>Wholesale</Tag>
 ```
 
+### Remove tag
+
+Use to allow merchants to add attributes to an object.
+
+```jsx
+<Tag onRemove={() => {}}>Wholesale</Tag>
```
Comment (kyledurand) [inline src/components/Tag/README.md:47]: 🙌

**Review by kyledurand — APPROVED** [review]
Awesome, thanks Andrew!

**Comment by alex-page** [comment]
@AndrewMusgrave  merge master into your branch some builds were broken. Sorry.

**Pull request merged**